docs/content/en/docs/reference/changelog.md

```diff
@@ -1,14 +1,31 @@
----
-linkTitle: "Changelog"
-weight: 55
----
 
 # Changelog
 
 ## [Unreleased]
 
-### Removed
+### Added
+
+- aws-iam-authenticator support
+- Allow the retention of node taints during an upgrade
 
+## v0.6.0 - 2021-10-29
+
+### Added
```

Comment:
Do we have tasks or concepts documented for any of these features? It would be great to be able to link readers back to our docs on getting started with each new feature.

Reply:
> Do we have tasks or concepts documented for any of these features? It would be great to be able to link readers back to our docs on getting started with each new feature.

I'll be writing up content related to the "management cluster". I know there are also docs on the "bring your own registry" feature and possibly for the upgrade improvements. I agree we should add links to those docs here once they are locked down.

Reply:
yes we do, I was deciding on whether we should add the issue or the prs' links. I will add the issue and docs links.